apps/desktop-wallet/src/App.tsx

```diff
@@ -54,6 +55,7 @@ const App = () => {
   const theme = useAppSelector((s) => s.global.theme)
 
   useAutoLock()
+  useAddressesDataPolling()
```

Comment:
Since our components do not anymore use the "signal" hook (which included the polling mechanism), we need to do polling somewhere globally.